device-types/Dell/PowerEdge-MX7000.yaml

```diff
@@ -2,9 +2,14 @@
 manufacturer: Dell
 model: PowerEdge MX7000
 slug: dell-poweredge-mx7000
-u_height: 7
+part_number: '5112814'
+u_height: 7.0
 is_full_depth: true
 subdevice_role: parent
+airflow: front-to-rear
+weight: 105.0
+weight_unit: lb
+comments: '[Product Page](https://www.dell.com/en-us/shop/ipovw/poweredge-mx7000)'
 power-ports:
```

Comment:
These could also be modules. (And the C14 power port being part of that module)
https://github.com/netbox-community/devicetype-library/issues/2473#issuecomment-2461604726

Reply:
According to: https://www.dell.com/support/manuals/en-us/poweredge-mx7000/pemx7000_ism_pub/back-view-of-the-enclosure?guid=guid-fb31612b-d051-46b0-9ec6-98e5ee1f9057&lang=en-us the PSU's are indeed modules but the power ports are on the chasis itself and not part of the PSU's